lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -19,61 +19,47 @@
                 )
             )}
         </p>
-    % if not credit_status["purchased"] and not credit_status["error"] :
-        <p class="message-copy credit-eligibility-msg">
-            ${_("You are now eligible for credit. <b>Congratulations!</b>")}
+    <div class="credit-action">
+        % if not credit_status["purchased"] and not credit_status["error"] :
+            <p class="message-copy credit-msg credit-eligibility-msg">
+                ## Translators: provider_name is the name of a credit provider or university (e.g. State University)
+                ${_("You are now eligible to purchase course credit from {provider_name} for this course. Click <strong>Get Credit</strong> to get started.").format(
```

Comment:
Click --> Select

We don't use "click" because that isn't true for all users (or even a majority of users soon, if mobile takes off like we're expecting it to).